lib/times.js

```diff
@@ -40,7 +40,9 @@ function create (types) {
 }
 
 var times = {
-  week: function ( ) { return create('weeks')(1); }
+	mins_msecs: function (value) { return value*60*1000; }
```

Comment:
Yeah, I when I wrote it I was just seeing how much of a diff it would make in the profiler.
What about timeMinsToMsecs  and timeMsecsToMins ?